src/wp-includes/media.php

```diff
@@ -1957,6 +1952,18 @@ function wp_img_tag_add_loading_optimization_attrs( $image, $context ) {
 	$height            = preg_match( '/ height=["\']([0-9]+)["\']/', $image, $match_height ) ? (int) $match_height[1] : null;
 	$loading_val       = preg_match( '/ loading=["\']([A-Za-z]+)["\']/', $image, $match_loading ) ? $match_loading[1] : null;
 	$fetchpriority_val = preg_match( '/ fetchpriority=["\']([A-Za-z]+)["\']/', $image, $match_fetchpriority ) ? $match_fetchpriority[1] : null;
+	$decoding_val      = preg_match( '/ decoding=["\']([A-Za-z]+)["\']/', $image, $match_decoding ) ? $match_decoding[1] : null;
+
+	$args = array(
+		'width'         => $width,
+		'height'        => $height,
+		'loading'       => $loading_val,
+		'fetchpriority' => $fetchpriority_val,
+	);
+
+	if ( $decoding_val ) {
+		$args['decoding'] = $decoding_val;
+	}
```

Comment:
This doesn't need to be conditional, we can always pass it since it's `null` if not present anyways, similar to how it is for the `loading` and `fetchpriority` values.
```suggestion
		'decoding'      => $decoding_val,
	);
```